docs/ref/templates/builtins.txt

```diff
@@ -439,9 +439,7 @@ Use of both ``and`` and ``or`` clauses within the same tag is allowed, with
 
     {% if athlete_list and coach_list or cheerleader_list %}
 
-will be interpreted like:
-
-.. code-block:: python
```

Comment:
Please revert this change, because `.. highlight:: html+django` is set at the top of this file, so a default highlighting is different for this file.

Reply:
Good catch. Thanks for finding that. Fixed in the latest revision.